### PR TITLE
[NIFI-12774] configure RPG

### PR DESCRIPTION
**Pull request by scottyaslan**
NIFI-12774

### CODE REVIEW

**Comment by mcgilman** [comment]
Will review...

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/state/flow/flow.effects.ts [hunk 1]
```diff
@@ -1250,6 +1255,54 @@ export class FlowEffects {
...
+                                request: {
+                                    components: [
+                                        {
+                                            id: request.entity.id,
+                                            componentType: request.type
+                                        }
+                                    ]
+                                }
+                            })
+                        );
```
Comment (mcgilman) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/state/flow/flow.effects.ts:1299]: We should conditionally dispatch this action like we do in other `EditXyz` dialogs if the user isn't routed elsewhere. We have an interceptor that may forward to an error page. If that happens we have a fallback to ensure that all dialogs are closed and the close payload will be `ROUTED`. If this were to occur we wouldn't want `selectComponents` to be dispatched.

**Review by mcgilman — COMMENTED** [review]
Nice work @scottyaslan! I've left a few comments below.

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/remote-process-group/edit-remote-process-group/edit-remote-process-group.component.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                            RAW
+                        </mat-option>
+                        <mat-option
+                            value="HTTP"
+                            nifiTooltip
+                            [tooltipComponentType]="TextTip"
+                            [tooltipInputData]="getOptionTipData('HTTP')"
+                            [delayClose]="false">
+                            HTTP
+                        </mat-option>
```
[the remark this answers is not in the document]
Comment (scottyaslan) [reply nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/remote-process-group/edit-remote-process-group/edit-remote-process-group.component.html:58]: ok I can remove these. I just saw they were present in current nifi so I added them here.

**Review by mcgilman — APPROVED** [review]
Thanks for the updates @scottyaslan! +1 will merge.

**Pull request merged**